first_django_pr/my_app/views.py

```diff
@@ -51,13 +48,12 @@ def course(request):
         return render(request, 'get_grade.html', context) #потім створю html
     else:
         return HttpResponse('Congratulations on the course method!')
-#Дописав
+
 def grade(request):
     user = User()
```

Comment:
Every time the grade view is used a new user instance is created is that behavior correct?

Reply:
Yes, that's right

Reply:
I don't know how to describe it.. on new login the user will get a random rating